data/categories.txt

```diff
@@ -40,7 +40,7 @@ category "outfit"
 	"Secondary Weapons"
 	"Ammunition"
 	"Systems"
-	"Power"
+	"Power Component"
```

Comment:
```suggestion
	"Power" # depreciated, should not be used anymore but kept for plugin compatibility
	"Power Component"
```
we could just do this

Reply:
should there be other change in the outfits file? what about the deprecated outfits?